### DISCUSSION TITLE
Max/MinLength error message not displayed on input type number

### DISCUSSION THREAD

**Discussion by caiofelixuc** [react-hook-form/react-hook-form#7398] [General]
Holla! Hello! its me again.

I think i found a bug, but im not sure.

I have an input, that has a max and min length, and its a required field. The validations error messages are displayed correctly, but when the input is type number the maxLength and minLength error message are not displayed.

I have a validation schema, passed right to the register api, that looks like this:
![image](https://user-images.githubusercontent.com/92024680/147767551-5cbfd723-87e7-42cb-86c3-5cb79420f776.png)

After the schema its mounted i pass it to the register api, like that: 
![image](https://user-images.githubusercontent.com/92024680/147767801-e59c3469-e62b-49fd-8851-23ad58733d9b.png)

The form is on 'onBlur' mode, so whenever i don´t type nothing, and blur the input, it displays the required error message and the max/minLength message (input type="text"):
![image](https://user-images.githubusercontent.com/92024680/147767931-169a3e42-c219-421d-89dc-257b82fc7f5c.png)
![image](https://user-images.githubusercontent.com/92024680/147768102-b3901583-7e5c-482d-9e53-68d2b1f957ed.png)

But if the input is type="number", it just do not display the error message for max/minLength, only the browser show a hover message, but that is not what i intend.

As you can see the input value is  lower than the min, so it should display the error, but it does not:
![image](https://user-images.githubusercontent.com/92024680/147768273-47cdccd1-c8b7-4d49-a798-94244af80931.png)

I hope its something on my end.

Hope to get an answer!

**Comment by bluebill1049** [comment]
use min and max for number related, minLength/maxLength for string

**Reply by duraipixel** [reply]
not coming
